BigQueryDataTransfer/synth.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+common = gcp.CommonTemplates()
+
+library = gapic.php_library(
+    service='bigquerydatatransfer',
+    version='v1',
+    config_path='/google/cloud/bigquery/datatransfer/artman_bigquerydatatransfer.yaml',
+    artman_output_name='google-cloud-bigquerydatatransfer-v1')
+
+# copy all src except partial veneer classes
```

Comment:
We may want to copy over the partials as well. For example, the `Experimental` portion of the "generated code warning" still remains in the partial. Perhaps we should only omit copying them over when we know the partial contains an implementation?

Maybe even a better step would be to copy over the partial implementation into the newly generated file? Is that something possible with the new tooling?

Reply:
I'll have the scripts copy the partial veneer classes too except for Iot, Redis, Spanner, Speech, and Vision (which contain implementations).

As for trying to copy the implementation out, it's possible (the script is python), but not currently available in the tool. The `replace` function is used to do basic string replacement like `sed` would.